samples/system-test/buckets.test.js

```diff
@@ -112,6 +112,25 @@ it("should disable a bucket's uniform bucket-level access", async () => {
   );
 });
 
+it('should configure a bucket cors', async () => {
+  execSync(`node configureBucketCors.js ${bucketName} 3600 
+  POST http://example.appspot.com content-type`);
```

Comment:
should `responseHeader` param also be passed?

Reply:
yes responseHeader's value is `content-type`